Detectors/GLOQC/include/GLOQC/MatchITSTPCQC.h

```diff
@@ -255,11 +263,16 @@ class MatchITSTPCQC
   void setMaxPtCut(float v) { mPtMaxCut = v; }
   void setEtaCut(float v) { mEtaCut = v; } // TODO: define 2 different values for min and max (*)
 
+  // K0
+  void setMaxK0Eta(float v) { mMaxEtaK0 = v; }
+  void setRefitK0(bool v) { mRefit = v; }
+  void setCutK0Mass(bool v) { mCutK0Mass = v; }
+
  private:
   std::shared_ptr<o2::globaltracking::DataRequest> mDataRequest;
   o2::globaltracking::RecoContainer mRecoCont;
   GID::mask_t mSrc = GID::getSourcesMask("ITS,TPC,ITS-TPC");
-  GID::mask_t mAllowedSources = GID::getSourcesMask("ITS,TPC,ITS-TPC");
+  GID::mask_t mAllowedSources = GID::getSourcesMask("ITS,TPC,ITS-TPC,ITS-TPC-TOF,TPC-TOF,TPC-TRD,ITS-TPC-TRD,TPC-TRD-TOF,ITS-TPC-TOF,ITS-TPC-TRD-TOF");
```

Comment:
I am not sure why all the sources are needed, since AFAIK the secondary-vertexing will only be run on `ITS,TPC,ITS-TPC` tracks.

Reply:
I let @shahor02 comment.

Reply:
Yes. 
If "all" should be used, I would prefer that, since it is less error-prone. Should I? 
What I am worried about is online... What do you think, @shahor02 ? Also for the various inputs, specified in the json (which you see in the JIRA ticket https://its.cern.ch/jira/browse/QC-1174 for now - I will commit them once the sw is finalized)... We will not have them all in online, will we? They are:
```
tofcluster:TOF/CLUSTERS/0;matchITSTPCTRDTOF:TOF/MTC_ITSTPCTRD/0;matchTPCTRDTOF/TOF/MTC_TPCTRD/0;matchITSTPCTOF:TOF/MTC_ITSTPC/0;matchTPCTOF:TOF/MTC_TPC/0;trackTPCTOF:TOF/TOFTRACKS_TPC/0;trigTPCTRD:TRD/TRGREC_TPC/0;trackTPCTRD:TRD/MATCH_TPC/0;trigITSTPCTRD:TRD/TRGREC_ITSTPC/0;trackITSTPCTRD:TRD/MATCH_ITSTPC/0;p2decay3body:GLO/PVTX_3BODYREFS/0;decay3body:GLO/DECAYS3BODY/0;decay3bodyIdx:GLO/DECAYS3BODY_IDX/0;p2cascs:GLO/PVTX_CASCREFS/0;cascs:GLO/CASCS/0;cascsIdx:GLO/CASCS_IDX/0;p2v0s:GLO/PVTX_V0REFS/0;v0s:GLO/V0S/0;v0sIdx:GLO/V0S_IDX/0;pvtx_tref:GLO/PVTX_TRMTCREFS/0;pvtx_trmtc:GLO/PVTX_TRMTC/0;pvtx:GLO/PVTX/0;trackITSTPC:GLO/TPCITS/0;trackITSTPCABREFS:GLO/TPCITSAB_REFS/0;trackITSTPCABCLID:GLO/TPCITSAB_CLID/0;trackTPC:TPC/TRACKS;trackTPCClRefs:TPC/CLUSREFS;trackITS:ITS/TRACKS/0;trackITSROF:ITS/ITSTrackROF/0;trackITSClIdx:ITS/TRACKCLSID/0;alpparITS:ITS/ALPIDEPARAM/0?lifetime=condition&ccdb-path=ITS/Config/AlpideParam;SVParam:GLO/SVPARAM/0?lifetime=condition&ccdb-path=GLO/Config/SVertexerParam
```

@f3sch : as soon as an ITSTPC track is matched to TRD, it becomes ITSTPCTRD, so we should not just put ITSTPC, for example.

Reply:
it is anyway and-ed with AllowedSources, so you can set `all`. For the online, the requested inputs should depend on the detectors and reco settings, e.g. like muon does: https://github.com/AliceO2Group/QualityControl/blob/cbad4017290d952dc90312a4e508bd5d1d63ef80/Modules/MUON/Common/src/TracksTask.cxx#L41-L57

Reply:
Ok, if I understand, they pass it from the json. But then, do they pick a different json depending on the detector composition? I mean, if there is no TRD, I need to change the inputs or don't I because I will have just empty messages? Will it crash if in online we don't have `GLO/PVTX_3BODYREFS/0`? (just as an example).

Chiara

Reply:
Hi @chiarazampolli, these are different thinks:
if you request an input which is not in the outputs of some upstream device, the topology generation will fail. 
So, for what concerns detector composition (reconstruction processes), the qc-workflow should have a built-in logic which loads json corresponding to available inputs, like https://github.com/AliceO2Group/O2DPG/blob/370b6e6fbbeca5bdd8d7a47816dd229a1277476c/DATA/production/qc-workflow.sh#L194-L200

For the different outputs from the SVertexer: all V0s, cascades and 3-body decay output are always available (hence the `GLO/PVTX_3BODYREFS/0`), even if e.g. only V0s are requested (what is not requested will be empty). 

In opposite, the strangeness tracking output is available only if it is requested.